cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -93,6 +95,9 @@ function UnmanagedClusterNetworkSettings(props: Partial<StepProps>) {
 
     const { umcState } = useContext(UmcStore);
 
+    const combinedNodeToHostPortMapping = `${umcState[STORE_SECTION_FORM].IP_ADDRESS}/${umcState[STORE_SECTION_FORM].NODE_PORT_MAPPING}:${umcState[STORE_SECTION_FORM].HOST_PORT_MAPPING}/${umcState[STORE_SECTION_FORM].CLUSTER_PROTOCOL}`;
```

Comment:
It looks like the format should be to have a : after the IP address, not a /; per Sean's feedback

'127.0.0.1:80:80/tcp'